perfkitbenchmarker/linux_benchmarks/mysql_service_benchmark.py

```diff
@@ -751,6 +791,18 @@ def Prepare(self, vm):
     logging.info('Set root password completed. Stdout:\n%s\nStderr:\n%s',
                  stdout, stderr)
 
+  def _ValidateSize(self, size):
+    """Validate flag for storage size and throw exception if invalid
```

Comment:
add "." to the end. The google linter complains if there is no punctuation at the end of the summary.